src/net.cpp

```diff
@@ -2908,3 +2961,13 @@ uint64_t CConnman::CalculateKeyedNetGroup(const CAddress& ad) const
 
     return GetDeterministicRandomizer(RANDOMIZER_ID_NETGROUP).Write(vchNetGroup.data(), vchNetGroup.size()).Finalize();
 }
+
+void CConnman::DumpAnchors() const
+{
+    // The node dumps:
+    // - current anchors; and
+    // - anchors that were known at startup but have not tried for connections yet
```

Comment:
slight wording clarification: s/have not tried for connections yet/we have not yet tried to connect to/

Reply:
Thanks! You know English is not my strong point :)
Going to fix it.

Reply:
Btw, https://github.com/bitcoin/bitcoin/pull/17428#discussion_r449650363

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-689127833).